corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views/query.js

```diff
@@ -384,6 +389,9 @@ hqDefine("cloudcare/js/formplayer/menus/views/query", function () {
 
         initGeocoders: function () {
             var self = this;
+            $(function () {
+                kissmetrics.track.event("Accessibility Tracking - Geocoder Seen in Case Seach");
```

Comment:
Does this definitely only fire when there are geocoders on the page? I don't see that limitation - it looks like this is called whenever the query list view is attached, so on every case search screen.

Reply:
I see, yes this should be moved.